### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'startswith' when logging in

### ISSUE THREAD

**Issue by keethesh** [adw0rd/instagrapi#124]
Getting the following error when running the script. It was working some hours ago but suddenly stopped. Was having the error with 1.5.5. Updated to 1.5.8, no luck. Also tried with different accounts, same result.

```Traceback (most recent call last):
  File "xxx/main.py", line 53, in __init__
    self.instagram.login_by_sessionid(session_id)
  File "xxx\lib\site-packages\instagrapi\mixins\auth.py", line 289, in login_by_sessionid
    user = self.user_info_v1(int(user_id))
  File "xxx\lib\site-packages\instagrapi\mixins\user.py", line 235, in user_info_v1
    return extract_user_v1(result["user"])
  File "xxx\lib\site-packages\instagrapi\extractors.py", line 153, in extract_user_v1
    return User(**data)
  File "pydantic\main.py", line 398, in pydantic.main.BaseModel.__init__
  File "pydantic\main.py", line 1034, in pydantic.main.validate_model
  File "pydantic\fields.py", line 716, in pydantic.fields.ModelField.validate
  File "pydantic\fields.py", line 913, in pydantic.fields.ModelField._apply_validators
  File "pydantic\class_validators.py", line 277, in pydantic.class_validators._generic_validator_cls.lambda3
  File "xxx\lib\site-packages\instagrapi\types.py", line 8, in validate_external_url
    if (v.startswith('http') and '://' in v) or isinstance(v, str):
AttributeError: 'NoneType' object has no attribute 'startswith'
python-BaseException
```

Thanks for providing us with such a wonderful library btw! :)

**Comment by riccardolunardi** [comment]
This is happening to me, too! It happened when I update to the last version. 1.5.5 works fine.
The error is slightly different, but it should be caused by the same thing if I'm not mistaken.

```
Traceback (most recent call last):
  File "/vb_monitor/instagram-vanguard/src/User.py", line 22, in __init__
    user_object = api.user_info_by_username(username).__dict__
  File "/vb_monitor/instagram-vanguard/src/MultiClient.py", line 73, in user_info_by_username
    return self.select_api().user_info_v1(username)
  File "/usr/local/lib/python3.8/dist-packages/instagrapi-1.5.8-py3.8.egg/instagrapi/mixins/user.py", line 235, in user_info_v1
  File "/usr/local/lib/python3.8/dist-packages/instagrapi-1.5.8-py3.8.egg/instagrapi/extractors.py", line 153, in extract_user_v1
  File "pydantic/main.py", line 360, in pydantic.main.BaseModel.__init__
  File "pydantic/main.py", line 986, in pydantic.main.validate_model
  File "pydantic/fields.py", line 572, in pydantic.fields.ModelField.validate
  File "pydantic/fields.py", line 745, in pydantic.fields.ModelField._apply_validators
  File "pydantic/class_validators.py", line 277, in pydantic.class_validators._generic_validator_cls.lambda3
  File "/usr/local/lib/python3.8/dist-packages/instagrapi-1.5.8-py3.8.egg/instagrapi/types.py", line 8, in validate_external_url
    # iterator.  Don't check the type!  Use hasattr to check for both
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by keethesh** [comment]
I'm setting v to an empty string before the check, but just returning an empty string on the first line of `validate_external_url` will work too, as a temporary fix until @adw0rd fixes that.

**Comment by adw0rd** [comment]
@keethesh @riccardolunardi fixed 1.5.9
